src/db/providers.go

```diff
@@ -43,3 +46,13 @@ func ProvideClickhouseDB(dbUrl string, config *gorm.Config) *gorm.DB {
 		return gorm.Open(clickhouse.Open(dbUrl), config)
 	})
 }
+
+func ProvideSqliteDB(dbUrl string, config *gorm.Config) *gorm.DB {
+	applogger.Warn("App: Using sqlite db")
+	return lo.Must(gorm.Open(sqlite.Open(dbUrl), config))
```

Comment:
not using `attemptToOpen` intentionally as this is for embedded db.
let me know if we need to use `attemptToOpen`.